### PR TITLE
Add samples index

### PR DESCRIPTION
**Pull request by lutzroeder**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,30 @@
...
 ## .NET MAUI Links
+
 * [.NET MAUI Website](https://dot.net/maui)
-* [.NET MAUI on GitHub](https://www.github.com/dotnet/maui)
-* [.NET MAUI Blog](https://devblogs.microsoft.com/dotnet/category/maui/)
-* [.NET MAUI Installation](https://docs.microsoft.com/dotnet/maui/get-started/first-app?pivots=windows)
 * [.NET MAUI Documenation](https://docs.microsoft.com/dotnet/maui)
-
+* [.NET MAUI Blog](https://devblogs.microsoft.com/dotnet/category/maui/)
+* [.NET MAUI GitHub](https://www.github.com/dotnet/maui)
```
Comment (rachelkang) [inline README.md:21]: nit: for consistency with the other links

```suggestion
* [.NET MAUI GitHub](https://github.com/dotnet/maui)

```

File: README.md [hunk 2]
```diff
@@ -1,22 +1,30 @@
...
 
 ## .NET Foundation
 
-.NET Interative Notebooks for C# is a [.NET Foundation](https://www.dotnetfoundation.org/projects) project.
-
 There are many .NET related projects on GitHub.
 
-- [.NET home repo](https://github.com/Microsoft/dotnet) - links to 100s of .NET projects, from Microsoft and the community.
+- [.NET home repo](https://github.com/Microsoft/dotnet) - links to 100s of .NET projects, from Microsoft and the community.
```
Comment (rachelkang) [inline README.md:27]: nit: I can see screen readers not reading `100s` as "hundreds"

```suggestion
- [.NET home repo](https://github.com/Microsoft/dotnet) - links to hundreds of .NET projects, from Microsoft and the community.

```

**Review by rachelkang — APPROVED** [review]
LGTM! yay samples :)

File: README.md [hunk 3]
```diff
@@ -1,22 +1,30 @@
 # .NET MAUI Samples
 
-This is the home for samples built with [.NET MAUI](https://dot.net/maui) that showcase how to build cross-platform apps for iOS, Android, macOS, and Windows from a single shared C# codebase.
+Samples built with .NET Multi-platform App UI (.NET MAUI).
+
+[.NET MAUI](https://dot.net/maui) is a cross-platform framework for creating mobile and desktop apps with C# and XAML. Using .NET MAUI, you can develop apps that can run on Android, iOS, iPadOS, macOS, and Windows from a single shared codebase.
+
+## .NET MAUI Samples Links
```
Comment (davidortinau) [inline README.md:7]: Let's break apart "official" samples from community samples. Move Rachel's and Javier's to a secondary section. And then list the individual samples at the top.

**Review by jamesmontemagno — APPROVED** [review]

**Pull request merged**